### ISSUE TITLE
Customizing layout

### ISSUE THREAD

**Issue by rjcds** [roest01/docker-speedtest-analyser#84]
Thanks for this project !

Any pointers on getting the default layout to show a 7 day trend (appears to default to today's data) ?

**Comment by jaxjexjox** [comment]
I don't understand how this is even working for you, when it is broken for everyone else?

**Comment by rjcds** [comment]
Yeah it was a bit flaky, so I moved on to https://github.com/sthuber90/docker-speedtest

**Much** better, and the developer is still interested :)
